### ISSUE TITLE
Following instructions results in empty course material page

### ISSUE THREAD

**Issue by kinnala** [apluslms/aplus-course-template#4]
I cloned this repository and followed the instructions. Sphinx is successfully compiling the html pages when running `./docker-compile.sh` as is evident by looking at the contents of `_build`. However, when I start A+ using `./docker-up.sh` and log into the course using `teacher` or `student` (didn't try other accounts) I can only see an empty page:

![image](https://github.com/user-attachments/assets/9082e964-bc84-4e83-baee-ee2bc453943c)

I have not modified the source code in any way but I'd expect the contents of the template `module01` to be visible here instead of an empty page.

I'm using Firefox on macOS and cannot observe anything suspicious in the developer console or in the terminal while launching the containers. It is not obvious to me how to proceed from here. EDIT: I tried earlier to run https://github.com/apluslms/aplus-manual and it seemed to work fine.

**Comment by kinnala** [comment]
This is what I see if I look directly at the contents of `_build` using Firefox:

<img width="831" alt="image" src="https://github.com/user-attachments/assets/e24fcf40-e054-4c14-a830-ca4054f7a17b">

Somehow this does not become visible inside A+.

**Comment by kinnala** [comment]
I learned that `grader` is responsible for reading the static data. So I wanted to check if that data is available in the correct place. I started a shell inside the container:
```
bc626a1816da   apluslms/run-mooc-grader:1.26   "/init run-django.sh…"   About a minute ago   Up About a minute   0.0.0.0:8080->8080/tcp             aplus-grader-1
gustaft1@G339CWYYKJ ~ % docker exec -it bc626 sh
```
and checked if the data is available inside the container:
```
# ls /srv/courses/default/_build/html
_sources  genindex.html  module01     search.html
_static   index.html	 objects.inv  searchindex.js
```
Everything is there so container volume mount is not the issue I think.

**Comment by kinnala** [comment]
I don't think I can easily debug this problem any further. It would require a better understanding of the underlying software architecture.

**Comment by kinnala** [comment]
It turns out I was looking at the incorrect course. "Def. Course" was not visible originally but now that I removed everything under `_build` and followed the instructions again it became visible.

It could be that I had run `docker-compile.sh` once before updating the submodule, which might've caused `index.yaml` under `_build/yaml` to be incorrect and Sphinx didn't recompile it due to caching. I have 
also previously run into many issues due to Sphinx caching too much.

In my own projects I have decided that it saves frustrations if in my compile scripts I simply remove everything under `_build` before running `make html` or such. This issue is however now resolved and sorry for bothering you.
